.github/workflows/windows.yml

```
@@ -17,87 +14,152 @@ concurrency:
-        if: matrix.build_type == 'Release'
-        shell: pwsh
-      - name: Run CMake (Debug)
-        run: cmake -S . -B build -G "Visual Studio 17 2022" -A ${{ matrix.architecture }} -DJSON_BuildTests=On -DJSON_FastTests=ON -DCMAKE_CXX_FLAGS="$env:flags"
-        if: matrix.build_type == 'Debug'
+
+      - name: Configure
         shell: pwsh
+        run: |
+          cmake -S . -B build `
+                -G "Visual Studio 17 2022" `
+                -A ${{ matrix.architecture }} `
+                -DJSON_BuildTests=ON `
+                -DCMAKE_CXX_FLAGS="$env:FLAGS"
+
       - name: Build
         run: cmake --build build --config ${{ matrix.build_type }} --parallel 10
+
       - name: Test
-        run: cd build ; ctest -j 10 -C ${{ matrix.build_type }} --output-on-failure
-
-#  clang:
-#    runs-on: windows-2025
-#    strategy:
-#      matrix:
-#        version: [11, 12, 13, 14, 15]
-#
-#    steps:
-#      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-#      - name: Install Clang
-#        run: curl -fsSL -o LLVM${{ matrix.version }}.exe https://github.com/llvm/llvm-project/releases/download/llvmorg-${{ matrix.version }}.0.0/LLVM-${{ matrix.version }}.0.0-win64.exe ; 7z x LLVM${{ matrix.version }}.exe -y -o"C:/Program Files/LLVM"
-#      - name: Run CMake
-#        run: cmake -S . -B build -DCMAKE_CXX_COMPILER="C:/Program Files/LLVM/bin/clang++.exe" -G"MinGW Makefiles" -DCMAKE_BUILD_TYPE=Debug -DJSON_BuildTests=On
-#      - name: Build
-#        run: cmake --build build --parallel 10
-#      - name: Test
-#        run: cd build ; ctest -j 10 -C Debug --exclude-regex "test-unicode" --output-on-failure
-#
-#  clang-cl-12:
-#    runs-on: windows-2025
-#    strategy:
-#      matrix:
-#        architecture: [Win32, x64]
-#
-#    steps:
-#      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-#      - name: Run CMake
-#        run: cmake -S . -B build -G "Visual Studio 16 2019" -A ${{ matrix.architecture }} -T ClangCL -DJSON_BuildTests=On
-#      - name: Build
-#        run: cmake --build build --config Debug --parallel 10
-#      - name: Test
-#        run: cd build ; ctest -j 10 -C Debug --exclude-regex "test-unicode" --output-on-failure
+        run: |
+          cd build
+          ctest -j 10 -C ${{ matrix.build_type }} --output-on-failure
+
+# ────────────────────────────────────────────
+# 4) LLVM/Clang matrix (Win-2022)
+# ────────────────────────────────────────────
+  clang:
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        llvm: [11, 12, 13, 14, 15]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Install LLVM ${{ matrix.llvm }}
+        run: |
+          curl -fsSL -o llvm.exe 
+            https://github.com/llvm/llvm-project/releases/download/llvmorg-${{ matrix.llvm }}.0.0/LLVM-${{ matrix.llvm }}.0.0-win64.exe
```

```suggestion
          curl -fsSL -o llvm.exe https://github.com/llvm/llvm-project/releases/download/llvmorg-${{ matrix.llvm }}.0.0/LLVM-${{ matrix.llvm }}.0.0-win64.exe
```

The curl command seems to break due to the newline.